tests/unit/components/about.spec.js

```diff
@@ -4,8 +4,9 @@ import Vuetify from 'vuetify'
 import VueX from 'vuex'
 import { cloneDeep } from 'lodash'
 import About from '@/views/About.vue'
-import { PEER_CONNECTED } from '../../../src/store/mutation-types'
-import { applicationStore } from '@/store/index.js'
+import { PEER_CONNECTED } from '@/store/mutation-types'
+import EdgeDeviceStore from '@/store/edge-device.js'
```

Comment:
@vickywane This should be triggering an error. `edge-device.js` exports `edgeDeviceModule` but this line imports `EdgeDeviceStore`. What gives?

Reply:
It wouldn't trigger an error as it's a **default export** from the `edge-device.js` file, and not a **named export**. 

I however understand that it's inconsistent and I'll fix that.